feature/media_details/presentation/src/main/java/com/sanaa/presentation/screen/actor/ActorViewModel.kt

```diff
@@ -130,11 +130,18 @@ class ActorViewModel @Inject constructor(
         updateState { copy(isLoading = true) }
         tryToExecute(
             callee = ::fetchActorDetails,
-            onSuccess = { updateState { copy(isLoading = false) } },
-            onError = { e -> updateState { copy(isLoading = false) } }
+                 onSuccess = {
+                updateState { copy(isLoading = false) }
+            },
+            onError = { e ->
```

Comment:
Please move this to a function and call it as referenced function something like:
```suggestion
            onError = ::onError